spec/support/active_record.rb

```diff
@@ -0,0 +1,19 @@
+ActiveRecord::Base.establish_connection(
+  adapter: "sqlite3",
+  database: "#{ File.expand_path(File.join(File.dirname(__FILE__), '..')) }/db/test.sqlite3"
+)
+
+ActiveRecord::Base.connection.execute("DROP TABLE IF EXISTS 'posts'")
```

Comment:
Totally unnecessary.

I could create a fake rails app and test/run against it, but I use such a small part of it. Just the ActiveRecord part, that it's not worth it.